src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java

```diff
@@ -185,4 +185,46 @@ private void queueDecisionIssue(Decision decision) {
             sessionEventService.createSessionEvent(decision.getOnlineHearing(), EventTypes.DECISION_ISSUED.getEventType());
         }
     }
+
+    @ApiOperation(value = "Reply to a decision", notes = "A POST request is used to reply to a decision")
+    @ApiResponses(value = {
+            @ApiResponse(code = 201, message = "Success", response = CreateDecisionResponse.class),
+            @ApiResponse(code = 401, message = "Unauthorised"),
+            @ApiResponse(code = 403, message = "Forbidden"),
+            @ApiResponse(code = 404, message = "Online hearing not found"),
+            @ApiResponse(code = 409, message = "Online hearing already contains a decision"),
+            @ApiResponse(code = 422, message = "Validation error")
+    })
+    @PostMapping(value = "/decisionreplies", consumes = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity replyToDecision(UriComponentsBuilder uriBuilder, @PathVariable UUID onlineHearingId, @Valid @RequestBody DecisionReplyRequest request) {
+
+        if(!request.getDecisionReply().equalsIgnoreCase(DecisionsStates.DECISIONS_ACCEPTED.getStateName())
+            && !request.getDecisionReply().equalsIgnoreCase(DecisionsStates.DECISIONS_REJECTED.getStateName())) {
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body("Decision reply field is not valid");
+        }
+
+        Optional<OnlineHearing> optionalOnlineHearing = onlineHearingService.retrieveOnlineHearing(onlineHearingId);
+        if (!optionalOnlineHearing.isPresent()) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Online hearing not found");
+        }
+
+        Optional<Decision> optionalDecision = decisionService.findByOnlineHearingId(onlineHearingId);
+        if (!optionalDecision.isPresent()) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Unable to find decision");
+        }
+
+        Decision decision = optionalDecision.get();
+        if(!decision.getDecisionstate().getState().equalsIgnoreCase(DecisionsStates.DECISION_ISSUED.getStateName())) {
+            return ResponseEntity.status(HttpStatus.NOT_FOUND).body("Decision must be issued before replying");
+        }
+
+        DecisionReply decisionReply = new DecisionReply();
+        DecisionReplyRequestMapper.map(request, decisionReply, decision);
+        decisionReply = decisionReplyService.createDecision(decisionReply);
```

Comment:
There can only be one reply per author, but you don't need to fix that in this branch because we're not sure where the IDAM reference is coming from yet.